src/ekobox/apply.py

```diff
@@ -48,6 +52,14 @@ def apply_pdf(
 CONTRACTION = "ajbk,bk"
 
 
+@dataclass
+class PdfResult:
+    """Helper class to collect PDF results."""
+
+    pdfs: Dict[Union[int, str], float]
```

Comment:
The union could be a `Label` or `Flavor` type.

Reply:
done in 9b0d8048